### PR TITLE
update(kubeproxy): Adding kill/inject command env for resource chaos

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]
LGTM. will test on the portal after merge on EKS.

File: workflows/kube-proxy-all/workflow.yaml [hunk 1]
```diff
@@ -110,6 +110,8 @@ spec:
                               value: '500'
                             - name: TOTAL_CHAOS_DURATION
                               value: '60' # in seconds
+                            - name: CHAOS_KILL_COMMAND
+                              value: 'kill $(find /proc -name exe -lname '*/dd' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}' |  head -n 1)'
```
Comment (ishangupta-ds) [inline workflows/kube-proxy-all/workflow.yaml]: Please use double quotes @ispeakc0de it is giving error while parsing with single quotes.

File: workflows/kube-proxy-all/workflow.yaml [hunk 2]
```diff
@@ -148,6 +150,8 @@ spec:
                               value: '1'
                             - name: TOTAL_CHAOS_DURATION
                               value: '60' # in seconds
+                            - name: CHAOS_KILL_COMMAND
+                              value: 'kill $(find /proc -name exe -lname '*/md5sum' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}' |  head -n 1)'
```
Comment (ishangupta-ds) [inline workflows/kube-proxy-all/workflow.yaml]: same here

File: workflows/kube-proxy-all/workflow_cron.yaml [hunk 3]
```diff
@@ -113,6 +113,8 @@ spec:
                                 value: '500'
                               - name: TOTAL_CHAOS_DURATION
                                 value: '60' # in seconds
+                              - name: CHAOS_KILL_COMMAND
+                                value: 'kill $(find /proc -name exe -lname '*/dd' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}' |  head -n 1)'
         container:
```
Comment (ishangupta-ds) [inline workflows/kube-proxy-all/workflow_cron.yaml]: same here

File: workflows/kube-proxy-all/workflow_cron.yaml [hunk 4]
```diff
@@ -151,6 +153,8 @@ spec:
                                 value: '1'
                               - name: TOTAL_CHAOS_DURATION
                                 value: '60' # in seconds
+                              - name: CHAOS_KILL_COMMAND
+                                value: 'kill $(find /proc -name exe -lname '*/md5sum' 2>&1 | grep -v 'Permission denied' | awk -F/ '{print $(NF-1)}' |  head -n 1)'
```
Comment (ishangupta-ds) [inline workflows/kube-proxy-all/workflow_cron.yaml]: same here

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
@ispeakc0de PTAL.

**Review by Jonsy13 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]
LGTM.

**Review by ksatchit — APPROVED** [review]

**Pull request merged**